services/frontend/src/scenes/Profile/Profile.scss

```diff
@@ -3,68 +3,50 @@
 @import "styles/_mixins.scss";
 
 .profile {
+    align-items: flex-start;
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
+    height: calc(100% - 80px);
+    justify-content: space-between;
+    padding: 0;
 
-    .profile-header {
+    .profile-content {
         align-items: center;
-        background-color: $color-primary;
         display: flex;
-        justify-content: space-between;
-        padding: $default-spacing $default-spacing*2;
+        flex-direction: column;
+        height: 100%;
+        overflow: auto;
+        padding: $default-spacing;
 
-        .logo-icon {
-            height: 32px;
-            width: 32px;
+        .profile-content-title {
+            color: $color-neutral-6;
         }
-
-        .profile-auth {
-            display: flex;
-
-            button {
-                background: none;
-                border: none;
-                font-weight: bold;
-                font-size: 16px;
-            }
-
-            .profile-login {
-                color: $color-light;
-                padding: $default-spacing/2 $default-spacing*1.5;
-            }
-
-            .profile-signup {
-                @include hover-background($color-light, darken($color-light, 5%));
-
-                border-radius: $default-border-radius;
-                color: $color-primary;
-                margin-left: $default-spacing * 2;
-                padding: $default-spacing/2 $default-spacing*1.5;
-            }
+        .profile-content-card {
+            margin: 16px $default-spacing*5 $default-spacing 0px;
+            padding: $default-spacing;
         }
     }
 
-    .profile-jumbotron {
+    .profile-sidebar {
         align-items: center;
         display: flex;
         flex-direction: column;
-        margin-top: 25vh;
-
-        .profile-tagline {
-            color: $color-text-secondary;
-            font-weight: bold;
-            margin-bottom: $default-spacing * 2;
-        }
-
-        .profile-create-profile {
-            @include hover-background($color-primary, $color-primary-hover);
-            @include fab-dropshadow();
+        justify-content: center;
+        margin: $default-spacing * 2;        
+
+        .profile-list-item {
+            border-left: 3px solid transparent;
+            padding: $default-spacing * 0.5 $default-spacing * 1.5;

```

Comment:
For reference, when a property has multiple attributes, and you're doing multiplications/divisions, I like to remove the spaces between the multiplication/division sign to keep the elements more visually grouped.

i.e. `padding: $default-spacing*0.5 $default-spacing*1.5;`

Up to you if you want to change it.

Reply:
Yeah that's more readable, I changed it by removing the spaces.